RecoLocalTracker/SubCollectionProducers/src/HITrackClusterRemover.cc

```diff
@@ -319,8 +319,7 @@ void HITrackClusterRemover::process(OmniClusterRef const &ocluster, SiStripDetId
     return;
   if (!fromTrack) {
     int clusCharge = 0;
-    for (std::vector<uint8_t>::const_iterator iAmp = cluster->amplitudes().begin(); iAmp != cluster->amplitudes().end();
-         ++iAmp) {
+    for (auto iAmp = cluster->amplitudes().begin(); iAmp != cluster->amplitudes().end(); ++iAmp) {
       clusCharge += *iAmp;
```

Comment:
```for (auto amp : cluster->amplitudes()) clusCharge += amp;``` or even std::accumulate

Reply:
this code is there since 20 years, not me to change